subsystem/day2_cluster_test.go

```diff
@@ -141,6 +141,7 @@ var _ = Describe("Day2 cluster tests", func() {
 		By("checking insufficient state state - one host, no connectivity check")
 		ips := hostutil.GenerateIPv4Addresses(2, defaultCIDRv4)
 		generateEssentialHostSteps(ctx, h, "h1host", ips[0])
+		generateDomain(ctx, h, "test-cluster", "")
```

Comment:
"test-cluster" consts?

Reply:
It's a var, depending on the test, the cluster name+Base domain defers between different tests.

Reply:
i mean that adding it as string in all the cases looks less pretty than creating 
var cluterName or using some const